api/capi/include/nntrainer_util.h

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (C) 2023 Samsung Electronics Co., Ltd. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * @file nntrainer_util.h
```

Comment:
check file name again and I wonder why do we need this.

Reply:
Actually, This script file is already existed but there are no explanation and code in this script.
But, we need to add the doxygen to this script, so I added it.

Reply:
If we don't need this script file, then I'll remove it.

Reply:
If there is no place to include this, then we could remove it.